### PR TITLE
corner esp

### PR DESCRIPTION
**Pull request by Laxybaxy**
just a simple corner esp for those who like that better than box-esp.
Color is configured with box-esp color

preview:
https://i.imgur.com/hn4zFpj.png

### CODE REVIEW

**Comment by deprale** [comment]
Cool, grenade ESP WHEN?

File: Osiris/Config.cpp [hunk 1]
```diff
@@ -158,6 +158,7 @@ void Config::load(size_t id) noexcept
             espConfig.eyeTracesColor[1] = espJson["Eye traces color"][1].asFloat();
             espConfig.eyeTracesColor[2] = espJson["Eye traces color"][2].asFloat();
         }
+        if (espJson.isMember("corner")) espConfig.corner = espJson["corner"].asBool();
```
Comment (danielkrupinski) [inline Osiris/Config.cpp]: JSON variable name should start with capital letter ("Corner" in this case).

File: Osiris/GUI.cpp [hunk 2]
```diff
@@ -570,6 +570,8 @@ void GUI::renderEspWindow() noexcept
         checkboxedColorPicker("Snaplines", &config.esp[currentItem].snaplines, config.esp[currentItem].snaplinesColor);
         ImGui::SameLine(spacing);
         checkboxedColorPicker("Box", &config.esp[currentItem].box, config.esp[currentItem].boxColor);
+        ImGui::SameLine(spacing*1.4f);
```
Comment (danielkrupinski) [inline Osiris/GUI.cpp]: Spacing before and after `*` would increase code readability.

File: Osiris/Config.cpp [hunk 3]
```diff
@@ -500,6 +501,7 @@ void Config::save(size_t id) const noexcept
         espJson["Eye traces color"][0] = espConfig.eyeTracesColor[0];
         espJson["Eye traces color"][1] = espConfig.eyeTracesColor[1];
         espJson["Eye traces color"][2] = espConfig.eyeTracesColor[2];
+        espJson["corner"] = espConfig.corner;
```
Comment (danielkrupinski) [inline Osiris/Config.cpp]: It needs to be "Corner" to save it properly.

**Comment by danielkrupinski** [comment]
Thanks!

**Pull request merged**